toolkits/global/packages/global-corporate-footer/README.md

```diff
@@ -1,29 +1,155 @@
+your Terminal:
+
+```
+npm install @springernature/global-corporate-footer
+```
+The Corporate footer component is designed to be the same across all brands. Because of this, it doesn't include any brand settings like some of the other components.
+
+Import the installed component code in your `scss` file:
 
 ```scss
-// Pick ONE of the brands below to include
 @import '@springernature/global-corporate-footer/scss/10-settings/default';
-@import '@springernature/global-corporate-footer/scss/10-settings/springer';
-@import '@springernature/global-corporate-footer/scss/10-settings/nature';
 
 // Include this with your other components
 @import '@springernature/global-corporate-footer/scss/50-components/corporate-footer';
 ```
 
-## Examples
+## Template
 
-### HTML
-```html
-<div class="c-corporate-footer">
-    <div class="u-container">
-        <img src="images/logos/sn-logo-white.svg" alt="Springer Nature" loading="lazy" width="140" height="14"/>
-        <p class="c-corporate-footer__legal" data-test="copyright">&copy; 2020 Springer Nature Limited</p>
-    </div>
-</div>
+Find a configurable template in the [`view` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/view). 
+
+You can see an example in the [`demo` folder](https://github.com/springernature/frontend-toolkits/tree/master/toolkits/global/packages/global-corporate-footer/demo).
+
+### Template configuration
+
+#### Using the Corporate footer with an existing footer
+
+The Corporate footer component doesn't include a `<footer>` region/landmark. This means you can use it together with - for example - `springer-nature-publisher` inside a common `<footer>`.
+
+#### Configuring the default or extended footer option
+
+The `demo`'s `context.json` file shows both the `default` and `extended` versions of the footer. 
+
+We use wrappers around each version to sandbox them in the demo - but you won't need these wrappers when you're using the component in your application.
+
+The actual configuration of the `corporate-footer` should look something like this example:
+
+```json
+{
+  "navigation": {
+  "links": [
+    {
+      "text": "Example link",
+      "url": "/url/to/example/link"
+    },
+    {
+      "text": "Example button",
+      "buttonProperties": "onlick=\"doSomething()\""
+    }
+  ]
+  },
+  "image": {
+    "src": "path/to/springer/nature/logo",
+    "alt": "Springer Nature",
+    "link": "https://www.springernature.com/"
+  },
+  "currentYear": 2022
+}
 ```
 
-Note that a `<footer>` region/landmark is not included. This allows you to combine this component with—for example—`springer-publisher-footer` inside a common `<footer>`.
+## Help improve this page
+
+If you’ve got a question, idea or suggestion about how to improve this component or guidance, post in the [#design-systems Slack channel](https://springernature.slack.com/archives/C75DHBTBP).
```

Comment:
```suggestion
If you’ve got a question, idea or suggestion about how to improve this component or guidance, post in the [#design-systems Slack channel](https://springernature.slack.com/archives/C75DHBTBP).

[info-npm]: https://www.npmjs.com/package/@springernature/global-corporate-footer
[badge-npm]: https://img.shields.io/npm/v/@springernature/global-corporate-footer.svg
```

This will make the badge at the top work!

Reply:
This will make the badge at the top work!